mmv1/products/tags/api.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Tags
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
```

Comment:
Is this in beta with a "v3" api?

Reply:
yep 😂 https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys

Reply:
well, I guess technically it's in "Preview"